draft/Contest_API.md

```diff
@@ -349,26 +349,24 @@ a change in a single object (submission, judgement, language, team,
 etc.) or full endpoint. The general format for events is:
 
 ```json
-{"contest_id": "<id>", "endpoint": "<endpoint>", "id": "<id>", "data": <JSON data for element> }
+{"type": "<type>", "id": "<id>", "data": <JSON data for element> }
 ```
 
 | Name        | Type   | Required? | Nullable? | Description
 | :---------- | :----- | :-------- | :-------- | :----------
-| contest\_id | string | yes       | no        | The contest id.
-| endpoint    | string | yes       | yes       | The API endpoint, i.e. type of contest object above. Can be used for filtering.
-| id          | string | yes       | yes       | The id of the object that changed.
-| data        | object | yes       | yes       | The data is the object that would be returned if calling the corresponding API endpoint at this time, i.e. an object or null for deletions.
+| type        | string | yes       | yes       | The type of contest object that changed. Can be used for filtering.
+| id          | string | yes       | yes       | The id of the object that changed, or null for the entire collection/singleton.
+| data        | object | yes       | yes       | The object that would be returned if calling the corresponding API endpoint at this time, i.e. an array, object, or null for deletions.
```

Comment:
I meant also the `Type` column. Should we say `object or array of objects`?

Reply:
Since it's referring to the type it seems natural enough to me that 'state' means the state changed and 'teams' means one or more teams changed (with details in the other attributes)... I can change it if you or others feel strongly about it though.

Reply:
Still think the type of this `data` field should be `object or array`.

Reply:
Fixed.